reference/misc/functions/defined.xml

```diff
@@ -11,10 +12,16 @@
   &reftitle.description;
   <methodsynopsis>
    <type>bool</type><methodname>defined</methodname>
-   <methodparam><type>string</type><parameter>nazwa</parameter></methodparam>
+   <methodparam><type>string</type><parameter>constant_name</parameter></methodparam>
   </methodsynopsis>
   <para>
-   Sprawdza czy stała o podanej nazwie istnieje i jest zdefiniowana.
+   Sprawdza, czy stała o nazwie podanej jako parametr <parameter>constant_name</parameter>
+   jest zdefiniowana.
+  </para>
+  <para>
+   Ta funkcja działa również z <link
+   linkend="language.oop5.constants">class constants</link> i <link
+   linkend="language.types.enumerations">enum cases</link>.
```

Comment:
"działa również ze stałymi klas" ....ale jak przetłumaczyć enum cases to faktycznie nie mam pojęcia. Niestety nie mogą to być "wartości enum" :thinking:

Reply:
Tak "elementy enum" byłoby super.

Reply:
```suggestion
   Ta funkcja działa również ze <link
   linkend="language.oop5.constants">stałymi klas</link> i <link
   linkend="language.types.enumerations">elementami enum</link>.
```